src/components/form/widgets/input-field.ts

```diff
@@ -50,9 +50,9 @@ export class InputField extends React.Component {
         if (event.detail || typeof event.detail === 'number') {
             value = event.detail;
         } else if (type === 'number') {
-            value = null;
+            value = undefined;
```

Comment:
It seems that setting it to `undefined` may have side effects 0259623

Reply:
Yes, we need to investigate more `null/undefined` issue. But when the `error.name === 'required' ` we could merge I guess. or?

Reply:
If we set it to `undefined` here, I think it will be impossible to remove a value (judging from the description in 0259623) 
It might also be that the issue is in the webclient and we need a better way to handle that issue there. Did you test what will happen on the card in the webclient with this set to `undefined` What happens when you remove a value and save the card? Will it be saved with the empty value or will the old value stick?

Reply:
yes, I have tested it in the webclient. It was possible to remove data but since these input fields are required it was not possible to save them as empty fields.

Reply:
Ok, then we need to test it with a field that is not required :slightly_smiling_face:

Reply:
yes, it's still possible to save the empty field (when it's not required)

Reply:
notes for the future: it was not possible to save an empty field when the `"type": "integer"`  (in case we changed null to undefined)